scripts/compactor_utils.py

```diff
@@ -0,0 +1,40 @@
+import subprocess
+import time
+import threading
+
```

Comment:
Shouldn't the `stop_threads` be defined here and instantiated with `False` at module level?

Reply:
Not necessary. The global keyword is sufficient when it's defined in local context to be used outside the method too.